Makefile

```diff
@@ -133,6 +133,10 @@ test-unit: generate ## Run unit tests only.
 test-integration: generate manifests build-ebpf ## Run integration tests.
 	EBPF_BUILD_DIR=$(EBPF_BUILD_DIR) ANTIMETAL_BPF_PATH=$(EBPF_BUILD_DIR) go test -tags integration ./... -v -timeout 60s
 
+.PHONY: test-coverage
+test-coverage: generate manifests build-ebpf ## Run tests with coverage. Usage: make test-coverage [TYPE=all|unit|integration|html|summary]
+	@$(ROOT)/scripts/test-with-coverage.sh $(TYPE)
+
```

Comment:
I think we could streamline this a bit and have the `test-unit` and `test-integration` targets by default print the coverage report to the coverage directory.

That way in `test-with-coverage.sh` we can just call `make test` to get all the coverage reports